eng/common/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/powershell
```

Comment:
Did you intend to put this under eng/common? If we do want it there we should move it somewhere not under the root of eng/common.

Reply:
Yes we should probably put it under just `eng/`